From fe0b01da63fb6506de1c6426aae77f7d5e2dec5d Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Wed, 11 Dec 2024 16:02:01 +0800
Subject: [PATCH 261/296] AOSCOS: staging: sb105x: drop upstream-removed
 STD_COM_FLAGS

Per commit 5691e03593cb ("tty: frv, remove unused serial macros"), the
`STD_COM_FLAGS' macro was not used anywhere and was then removed.

Follow this commit and remove the aforementioned macro.

Fixes: "AOSCOS: Revert "staging: sb105x: delete the driver""
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/staging/sb105x/sb_pci_mp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/sb105x/sb_pci_mp.c b/drivers/staging/sb105x/sb_pci_mp.c
index c5e1c39e7a7a..32f60de269cc 100644
--- a/drivers/staging/sb105x/sb_pci_mp.c
+++ b/drivers/staging/sb105x/sb_pci_mp.c
@@ -2808,7 +2808,7 @@ static void __init multi_init_ports(void)
 				osc = 0;
 			for(j=0;j<osc;j++)
 				mtpt->port.uartclk *= 2;
-			mtpt->port.flags    |= STD_COM_FLAGS | UPF_SHARE_IRQ ;
+			mtpt->port.flags    |= UPF_SHARE_IRQ ;
 			mtpt->port.iotype   = UPIO_PORT;
 			mtpt->port.ops      = &multi_pops;
 
-- 
2.48.1

